src/actions/passphrase.js

```diff
@@ -14,14 +14,15 @@ export const UPDATE_PASSPHRASE_2FA_2 = 'UPDATE_PASSPHRASE_2FA_2'
 export const UPDATE_PASSPHRASE_2FA_2_SUCCESS = 'UPDATE_PASSPHRASE_2FA_2_SUCCESS'
 export const UPDATE_PASSPHRASE_2FA_2_FAILURE = 'UPDATE_PASSPHRASE_2FA_2_FAILURE'
 
-export const updatePassphrase = (current, newVal) => {
+export const updatePassphrase = (current, newVal, callback) => {
   return async (dispatch, getState) => {
     dispatch({ type: UPDATE_PASSPHRASE })
     return cozyFetch('PUT', '/settings/passphrase', {
       'current_passphrase': current,
       'new_passphrase': newVal
     }).then(instance => {
       dispatch({ type: UPDATE_PASSPHRASE_SUCCESS })
+      callback()
```

Comment:
Why using a callback as we are already in an asynchronous context, which the role is specially to avoid callbacks ?